eng/helix/content/RunTests/RunTests.csproj

```diff
@@ -4,6 +4,7 @@
     <OutputType>Exe</OutputType>
     <TargetFramework>net7.0</TargetFramework>
     <DefineConstants Condition=" '$(InstallPlaywright)' == 'true' ">$(DefineConstants);INSTALLPLAYWRIGHT</DefineConstants>
+    <NoWarn>$(NoWarn);CA2007</NoWarn>
```

Comment:
why not add a `ConfigureAwait(...)`❔

Reply:
Figured it didn't really matter in this context (tests). 

We do it elsewhere, as well:

https://github.com/dotnet/aspnetcore/blob/f041d49f3320fd0fb70babe35a861a9035e48e7e/eng/targets/CSharp.Common.targets#L37-L41

Reply:
Shouldn't need to duplicate that 😸

Reply:
> Would still prefer you remove the duplication around `CA2007`

https://github.com/dotnet/aspnetcore/pull/41311/commits/3858e46da674aee597ae39c1925680b6ac5c3eb8